certs/service.go

```diff
@@ -186,7 +185,7 @@ func (cs *certsService) ViewCert(ctx context.Context, token, serialID string) (C
 
 	cert, err := cs.certsRepo.RetrieveBySerial(ctx, u.GetId(), serialID)
 	if err != nil {
-		return Cert{}, errors.Wrap(repoerr.ErrNotFound, err)
+		return Cert{}, errors.Wrap(svcerr.ErrNotFound, err)
```

Comment:
This should be. Also check other cases where we wrap `ErrNotFound` when we fail to view an entity from the DB.
```suggestion
		return Cert{}, errors.Wrap(svcerr.ErrViewEntity, err)
```